app/components/NavHeader.tsx

```diff
@@ -4,8 +4,8 @@ import { FaEarthAsia } from 'react-icons/fa6';
 
 const NavHeader = () => {
   return (
-    <div className="flex items-center h-[60px] bg-purple-heart-950 gap-4 px-4 py-4 text-white text-lg font-medium">
-      <FaEarthAsia size="30px" color="white" />
+    <div className="flex items-center h-[60px] bg-orange-50 gap-4 px-4 py-4 text-black-700 text-lg font-medium">
+      <FaEarthAsia size="30px" color="#e12700" />
```

Comment:
Hard to go wrong with orange/blue 

Define a blue scale in the tailwind config and try this out as blue